app/javascript/controllers/ontologies_selector_controller.js

```diff
@@ -0,0 +1,74 @@
+import {Controller} from "@hotwired/stimulus"
+
+export default class extends Controller {
```

Comment:
here is you can't extract the count logic to a controller, and re-use it the browse page ? as the logic is the same it count for a block the selected items and display the count in the title